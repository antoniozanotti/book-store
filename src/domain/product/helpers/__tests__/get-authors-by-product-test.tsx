import { productWithPagEandBookMock } from "../../__tests__/product-with-page-and-book-mock";
import { productsWithPagEandBookMock } from "../../__tests__/products-with-page-and-book-mock";
import { getAuthorsByProduct } from "../get-authors-by-product";

describe("getAuthorsByProduct", () => {
  it("should return authors by product", async () => {
    const authors = getAuthorsByProduct(productsWithPagEandBookMock[0]);
    expect(authors).toMatchSnapshot();
  });
});
