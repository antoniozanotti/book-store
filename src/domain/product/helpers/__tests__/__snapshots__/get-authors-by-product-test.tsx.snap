// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getAuthorsByProduct should return authors by product 1`] = `"Don Miguel Ruiz"`;
